storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/design/GetOperationDesignTest.java

```diff
@@ -40,4 +40,17 @@ public void getListOfObjectsBlocking() {
                 .prepare()
                 .executeAsBlocking();
     }
+
+    @Test
+    public void geObjectBlocking() {
```

Comment:
Not sure if I get your comment.
As about design tests - they are pretty useful as a proof of concept